.github/workflows/flink.yaml

```diff
@@ -29,10 +29,9 @@ jobs:
       matrix:
         python-version: [ "3.9", "3.10", "3.11" ]
         recipes-version: [
-            "pangeo-forge-recipes==0.9.4",
+            "pangeo-forge-recipes==0.10.0",
```

Comment:
@ranchodeluxe generally I like deprecating 0.9.4 support (#144) but for my understanding, is there a reason this is part of this PR?

Reply:
And thanks for the lightening-fast turnaround on this!

Reply:
oh, haha, we can change it back so you can do the deprecation sorry (just thought I'd add it in)